src/layout/AppLayout.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="app-layout">
+    <div class="app-layout__sidebar"><slot name="sidebar"></slot></div>
+    <main class="app-layout__main"><slot></slot></main>
+  </div>
+</template>
+
+<style>
```

Comment:
could use SCSS here too.

Reply:
Yep, I might want to remove our dependency on scss, as we are not using many of it's features, and there is one less dependency to worry about. At the moment we are locked on node v. 14.16 because of Sass runtime problems with later node versions.